desktop/cmp/dash/canvas/DashCanvasModel.js

```diff
@@ -168,6 +170,14 @@ export class DashCanvasModel extends HoistModel {
             track: () => [this.viewState, this.layout],
             run: () => this.publishState()
         });
+
+        this.addReaction({
+            when: () => this.ref.current,
+            run: () => {
+                const {current: node} = this.ref;
```

Comment:
find the destructure with aliasing a little ponderous -- would like to discuss this vs.
`const node = this.ref.current;`

Reply:
also, in this case, node can just be the argument to the run function.

Reply:
Sounds good- I see what you're saying.  I think I tried using `node` as the argument to the `run` function and it not working, but will double-check.  I assumed that maybe it was only passed to `run` when using `track` (and not `when`)